t/test-rest-api.php

```diff
@@ -273,6 +294,31 @@ function test_update_post_state() {
 
 	/**
 	 * Integration test
+	 * Test accessing the get entries endpoint
+	 */
+	function test_endpoint_get_entries() {
+		// Insert 1 entry
+		$this->insert_entries();
+
+		// A time window with entries
+		$start_time = strtotime( '-1 hour' );
+		$end_time   = strtotime( '+1 hour' );
+
+		// Try to access the endpoint
+		$request = new WP_REST_Request( 'GET', self::ENDPOINT_BASE . '/1/entries/' . $start_time . '/' . $end_time . '/');
+		$response = $this->server->dispatch( $request );
+
+		// Assert successful response
+		$this->assertEquals( 200, $response->get_status() );
+
+		$entries = $response->get_data();
+
+		// The array should contain 1 entry
+		$this->assertCount( 1, $entries['entries'] );
```

Comment:
Could we do some deeper assertions here? (i.e. instead of just checking that we have one entry, that the entry data matches what we expect?)